### DISCUSSION TITLE
mais uma criação da feiticeira do CSS

### DISCUSSION THREAD

**Discussion by leandroruel** [frontendbr/forum#1593] [Dicas]
Vendo mais essa criação, fico me perguntando se eu realmente sei CSS, impostor bateu forte agora.
https://twitter.com/cyanharlow/status/1191264717706149890

site: https://diana-adrianne.com/purecss-lace/

**Comment by felipefialho** [comment]
Mano, o trabalho da Diana Smith é formidável, sou um fã de arte feita com CSS e fico realmente muito feliz observando as criações dela.

Mas você não deveria se abater por "síndrome do impostor" por conta disso, acredito que poucas pessoas no MUNDO seriam capazes de fazer alguma coisa desse tipo, e mesmo assim, é uma forma de arte e de uso muito especifico.

Foca no uso do CSS no dia-a-dia, ou se de fato se interessar por criar coisas desse tipo, invista tempo em aprender e desenvolver esse lado 😄

**Comment by diglopes** [comment]
Não a conhecia e fiquei abismado com o que ela faz, é muito absurdo mano!

**Comment by Juuhz** [comment]
Bizarro!

**Comment by gabrielclaudino** [comment]
meu deus
